apps/ethereum_jsonrpc/lib/ethereum_jsonrpc/transaction.ex

```diff
@@ -311,6 +311,11 @@ defmodule EthereumJSONRPC.Transaction do
     {key, quantity_to_integer(quantity)}
   end
 
+  # as always ganache has it's own vision on JSON RPC standard
+  defp entry_to_elixir({key, quantity}) when key in ~w(r s v) and quantity == nil do
```

Comment:
```suggestion
  defp entry_to_elixir({key, nil}) when key in ~w(r s v) do
```

Don't use a variable and guard `when variable == value` for a fixed `value`.  Match it directly in the pattern.